### ISSUE TITLE
Longer placeholders are cut-off in some cases

### ISSUE THREAD

**Issue by Droxx** [shentao/vue-multiselect#757]
### Reproduction Link
https://jsfiddle.net/jqofkzxc/3633/

### Steps to reproduce
Click on the multi-select tool

### Expected behaviour
Placeholder "This is a very long placeholder to demonstrate the issue" should be displayed

### Actual behaviour
Only "This is a very long placeh" is displayed

**Comment by stevebauman** [comment]
Why was this closed? This still occurs in the most recent 2.1.3 release.

**Comment by shentao** [comment]
You're right! The bot closed (the bot is now removed). Sorry for the trouble.

**Comment by stevebauman** [comment]
No problem! Thanks for re-opening so quicky :smile:

I love this package btw, use it in multiple projects.

**Comment by farrelmahaztra** [comment]
The entire placeholder shows for me when disabling "width: auto" on the innermost input tag, that might be helpful (2.1.4).
